src/lib/genesis_ledger_helper/genesis_ledger_helper.ml

```diff
@@ -916,8 +993,15 @@ let inputs_from_config_file ?(genesis_dir = Cache_dir.autogen_path) ~logger
           "Proof level %s is not compatible with compile-time proof level %s"
           (str proof_level) (str compiled)
   in
+  let s3_hash =
```

Comment:
A bit of code duplication with lines above, consider introducing a helper function